darwin/future/tests/core/items/test_set_priority.py

```diff
@@ -0,0 +1,93 @@
+from unittest.mock import Mock
+
+import pytest
+
+from darwin.exceptions import DarwinException
+from darwin.future.core.client import ClientCore
+from darwin.future.core.items.set_priority import set_item_priority
+
+
+def test_set_item_priority() -> None:
+    # Create a mock API client
+    api_client = Mock(spec=ClientCore)
```

Comment:
We do have the response library for this, although not at issue by any means.

Reply:
Yeah, actually I'll rewrite with that in order to maintain our test style